lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -346,12 +346,21 @@
           <h2 class="heading -emphasized -inverse"><span><?php print t('Step 4: Prove It'); ?></span></h2>
         </div>
       </div>
+
       <div class="container__row">
           <div class="container__block">
             <h3 class="inline-alt-text-color"><?php print t('Pics or It Didn&rsquo;t Happen'); ?></h3>
             <?php if (isset($reportback_copy)): ?>
               <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
             <?php endif; ?>
+
+            <h3 class="inline-alt-text-color"><?php print t('Helpful Tips'); ?></h3>
```

Comment:
for this stuffs, we added a new issue to translate a bunch of the new stuff we added. #6970
I'm not sure if it's a priority